src/ui-actions/participant-displayer-actions.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import { t, getBridgedFunctions } from '@zextras/carbonio-shell-ui';
+
+import { mailToSharedFunction } from '../integrations/shared-functions';
+import { Participant } from '../types';
+
+export const copyEmailToClipboard = (email: string): void => {
+	navigator.clipboard.writeText(email).then(() => {
+		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+		// @ts-ignore
+		getBridgedFunctions().createSnackbar({
```

Comment:
createSnackbar should be received as a prop (and generated by the DS) as the shell is soon to discard the export of the snackbar manager